web_src/less/_review.less

```diff
@@ -166,10 +166,7 @@
     .button {
       width: 100%;
       margin: 0 !important;
-
-      &:not(:last-child) {
-        margin-bottom: .75rem !important;
-      }
```

Comment:
Won't that break something else?

Reply:
Well I don't hope so, it was originally added in #17321, which seems like this selector(`.comment-code-cloud .button`) is already specific to these buttons

Reply:
Did a check and the only other affected button that will now receive `margin-bottom` is the cancel button for the reply comment box:

![image](https://user-images.githubusercontent.com/25481501/195954228-569044f0-f616-4cb7-a2fe-74c7ebcbd40a.png)